internal/olm/installer/client.go

```diff
@@ -183,6 +187,15 @@ func (c Client) getResources(ctx context.Context, version string) ([]unstructure
 }
 
 func (c Client) getCRDs(ctx context.Context, version string) ([]unstructured.Unstructured, error) {
+	if isAvailableLocally(version) {
```

Comment:
Just run this conditional logic once in `getResources()`, and default to `getCRDs()/getOLM()` if `isAvailableLocally(version) == false`.